Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -1641,6 +1656,60 @@ def checkpoint_show_task_command(client: Client, task_id: str) -> CommandResults
     return command_results
 
 
+def checkpoint_add_objects_batch_command(client: Client, object_type: str, ipaddress, name):
+    printable_result = {}
+    readable_output = ''
+
+    ipaddress = argToList(ipaddress, ',')
+    name = argToList(name, ',')
+    add_list = []
+    for ip, n in zip(ipaddress, name):
+        tmp_dict = {'name': n, 'ip-address': ip}
+        add_list.append(tmp_dict)
+
+    result = current_result = client.add_objects_batch(object_type, add_list)
+
+    if result:
+        printable_result = {'task-id': result.get('task-id')}
```

Comment:
The context outputs here are not in the correct format, see https://xsoar.pan.dev/docs/integrations/context-and-outputs/

Also, how do we expect the user to use this context data?